### DISCUSSION TITLE
Crazyswarm crashes throwing an instance of 'XmlRpc::XmlRpcException'

### DISCUSSION THREAD

**Discussion by DARCLab** [USC-ACTLab/crazyswarm#637] [Q&A]
I am trying to connect/fly one Crazyflie using hover_swarm.launch with OptiTrack motion capture system. I can successfully communicate with the CF. 

crazyswarm server crashes with the following error message: 
[crazyflieTerminalOutput.txt](https://github.com/USC-ACTLab/crazyswarm/files/8558652/crazyflieTerminalOutput.txt)
 
Here is my hover_swarm launch file
[hover_swarm.txt](https://github.com/USC-ACTLab/crazyswarm/files/8558656/hover_swarm.txt)

The main motive parameters are:
Broadcast Frame: On
Local interface: 192.168.1.3
Labeled Markers: On
Unlabeled Markers: Off
Asset Markers: Off
Rigid Bodies: On
Command Port: 1510
Data Port: 1511
Multicast interface: 239.255.42.99

mocap_helper.launch works publishing out the points and rigid body of my CF:
frame 110:
    points:
      "0": [-2.59462,-0.147171,0.0497183]
      "1": [-2.56597,-0.115718,0.0493726]
      "2": [-2.59738,-0.0872367,0.050428]
      "3": [-2.6086,-0.130656,0.0505769]
      "4": [-2.59601,-0.117168,0.0496659]
    rigidBodies:
      "1":
       position: [-2.59252, -0.11959, 0.0499524]
       rotation: [-0.997245, -0.0326443, 0.00419566, -0.0664749]

And have also ran the NatNetSDKCrossplatform and got the following result:

Begin Packet

NatNetVersion 128 150 152 0
Message ID  : 7 NAT_FRAMEOFDATA
Packet Size : 246
MoCap Frame Begin

Frame # : 16611895
Marker Set Count :   0
Unlabeled Markers Count : 0
Rigid Body Count :   1
  RB:   0 ID :   1
    pos: [-2.59,-0.12,0.05]
    ori: [-0.03,0.00,-0.07,-1.00]
    Mean marker err: 0.00
    Tracking Valid: True
Skeleton Count : 0
Labeled Marker Count : 5
  ID  : [MarkerID: 113] [ModelID: 0]
    pos : [-2.59,-0.15,0.05]
    size: [0.01]
    err:  [0.00]
  ID  : [MarkerID: 114] [ModelID: 0]
    pos : [-2.57,-0.12,0.05]
    size: [0.01]
    err:  [0.00]
  ID  : [MarkerID: 115] [ModelID: 0]
    pos : [-2.60,-0.09,0.05]
    size: [0.01]
    err:  [0.00]
  ID  : [MarkerID: 116] [ModelID: 0]
    pos : [-2.61,-0.13,0.05]
    size: [0.01]
    err:  [0.00]
  ID  : [MarkerID: 117] [ModelID: 0]
    pos : [-2.60,-0.12,0.05]
    size: [0.01]
    err:  [0.00]
Force Plate Count: 0
Device Count: 0
Timestamp : 138432.458
Mid-exposure timestamp         : 0
Camera data received timestamp : 276622909605
Transmit timestamp             : 276622934611
MoCap Frame End

End Packet

Thank you in advance!

Regards DARCLab

**Comment by whoenig** [comment]
This crash usually means that there is an issue with your crazyflies.yaml file (or crazyflieTypes.yaml, in case you edited that one). Could you share those files?
